app/src/main/res/values/strings.xml

```diff
@@ -142,6 +142,7 @@
 
   <string name="since_admission">Since admission</string>
   <string name="since_symptom_onset">Since symptom onset</string>
+  <string name="weight">Weight</string>
```

Comment:
Should probably be Weight (kg).

Long term, I think we'd want to load these strings from the concept name in the profile.